controls/radslider/declaration.md

```diff
@@ -49,5 +52,22 @@ Code-behind declaration:
 	slider.SmallChange = 1
 {{endregion}}
 
+#### Figure 1: Result from Examples 1 and 2
+![RadSlider](images/radslider_gettingstarted.png)
+
+## Key Properties
+
+The __RadSlider__ control derives from the [DoubleRangeBase](https://docs.telerik.com/devtools/silverlight/api/html/t_telerik_windows_controls_doublerangebase.htm) class which in turn derives from the [RangeBase](https://msdn.microsoft.com/en-us/library/system.windows.controls.primitives.rangebase(v=vs.110).aspx) class. In addition to the inherited properties, these are some of the properties the RadSlider defines:
```

Comment:
I don't see the need of telling that the control derives from RangeBase. This is used in many articles in our doco but I don't think that brings any value to the reader.